packages/login/src/PasswordUpdateApp.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import {appFactory, errorLogging, externalEvents} from 'tocco-app-extensions'
+import {consoleLogger} from 'tocco-util'
+
+import LoginContainer from './containers/LoginContainer'
+import * as passwordUpdate from './modules/passwordUpdate/dialog/actions'
+import loginReducers, {sagas} from './modules/reducers'
+
+const packageName = 'login'
```

Comment:
I would use 'passwort-update' here and then you could use `packageName` on line 48